core/storage/user/gae_models_test.py

```diff
@@ -242,3 +242,51 @@ def test_get_multi_failure(self):
             SKILL_IDS)
 
         self.assertEqual(retrieved_object, [None, None])
+
+
+class UserContributionsScoringModelTests(test_utils.GenericTestBase):
+    """Tests for UserContributionScoringModel."""
+
+    def test_create_model(self):
+        user_models.UserContributionScoringModel.create('user1', 'category1', 1)
+        scoreModels = (user_models.UserContributionScoringModel
+                       .get_all_scores_of_user('user1'))
+        self.assertEqual(len(scoreModels), 1)
+        self.assertEqual(scoreModels[0].id, 'category1.user1')
+        self.assertEqual(scoreModels[0].user_id, 'user1')
+        self.assertEqual(scoreModels[0].score_category, 'category1')
+        self.assertEqual(scoreModels[0].score, 1)
+
+    def test_create_failure(self):
```

Comment:
It may be useful to indicate in this test name that the failure is due to the same scoring model existing.

Reply:
Done